src/InternPool.zig

```diff
@@ -5968,8 +5968,22 @@ pub const FuncAnalysis = packed struct(u32) {
     /// True if this function has an inferred error set.
     inferred_error_set: bool,
     disable_instrumentation: bool,
-
-    _: u24 = 0,
+    disable_intrinsics: bool,
+
+    _: u23 = 0,
+
+    pub const State = enum(u2) {
+        /// The runtime function has never been referenced.
+        /// As such, it has never been analyzed, nor is it queued for analysis.
+        unreferenced,
+        /// The runtime function has been referenced, but has not yet been analyzed.
+        /// Its semantic analysis is queued.
+        queued,
+        /// The runtime function has been (or is currently being) semantically analyzed.
+        /// To know if analysis succeeded, consult `zcu.[transitive_]failed_analysis`.
+        /// To know if analysis is up-to-date, consult `zcu.[potentially_]outdated`.
+        analyzed,
+    };
```

Comment:
not sure what's going on here, is this a merge conflict?

Reply:
Huh, yeah, looks like I messed up a rebase at some point.